tests/fixtures/issues/issue-618.php

```diff
@@ -2336,11 +2336,11 @@
     'issue-618-AS' => [
         'ua' => 'Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US) AppleWebKit/532.5 (KHTML, like Gecko) Chrome/4.1.249.1025 Safari/532.5',
         'properties' => [
-            'Comment' => 'Comodo Dragon 4.1',
```

Comment:
As mentioned in #1448, there **was** a Chrome 4.1 release on Windows, so I removed that version from the Dragon pattern so that the Chrome patterns can pick it up.